Spawn docker build relative to the Dockerfile (#9729)
cloudflare/workers-sdk: .changeset/clever-regions-build.md

---
"wrangler": patch
---

> Correctly set docker build context during deploy when `image_build_context` is not provided

Reviewer: - Clarify "Correctly" 
- Looks to me that it should be "build" rather than "deploy"?

```suggestion
Set docker build context to the Dockerfile directory when `image_build_context` is not explicitly provided
```

Reviewer 2: set it one layer down in constructBuildCommand 👍
